orga/tests/unit/components/auth/login-form_test.js

```diff
@@ -30,5 +36,65 @@ module('Unit | Component | Routes | login-form', (hooks) => {
       // then
       assert.ok(_authenticateStub.calledWith(sinon.match.any, expectedEmail));
     });
+
+    module('When there is no invitation', () => {
+      test('should not accept organization invitation when parameters are invalid', function (assert) {
+        // given
+        component.email = '';
+        component.password = 'pix123';
+        component.args.isWithInvitation = true;
```

Comment:
quand je lis `component.args.isWithInvitation = true;` sans trop savoir ce que ça fait, je me demande si le module doit s'appeller `module('When there is no invitation'` car avec un true j'ai tendance à penser qu'il y a une invitation

Reply:
il y a eu une inversion dans le nom des modules, je vais inverser ça :)